### ISSUE TITLE
Received invalid creationParameter when calling client.llm.load(modelPath)

### ISSUE THREAD

**Issue by ireicht** [lmstudio-ai/lmstudio.js#45]
So far I could run lmstudio.js lmstudio/sdk command "`await client.llm.load(modelPath);`" without any issue. I recently updated LM Studio from 0.2.25 to version 0.2.26 and lmstudio.js lmstudio/sdk from 0.0.3 to 0.0.12. When calling js command "`await client.llm.load(modelPath)`;" it throws this error:

```
W [LMStudioClient][LLM][ClientPort] Received communication warning from the server: Received invalid creationParameter for channel, endpointName = loadModel, creationParameter = {"path":"lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF/Meta-Llama-3-8B-Instruct-Q4_K_M.gguf","config":{"gpuOffload":1},"noHup":false}. Zod error:

- creationParameter.loadConfigStack: Required

This is usually caused by communication protocol incompatibility. Please make sure you are using the up-to-date versions of the SDK and LM Studio.

Note: This warning was received from the server and is printed on the client for convenience.

```
also adding a configuration parameter resulted into the same error:
```
client.llm.load(modelPath, {
            config: { gpuOffload: 1.0 }
        });
```

However other functions such as "client.system.listDownloadedModels()" still work fine. Only client.llm.load() has issues.

**Comment by yagil** [comment]
Thanks @ireicht! Which operating system are you on?

**Comment by brianpollack** [comment]
I think this is a duplicate of issue 42.   Same issue on Mac w/ node 18.20.0 using llm.load and llm.get.  Event the test cases from the README no longer work.  I haven't had time myself to look at it.

**Comment by yagil** [comment]
We found the issue. Will be solved in LM Studio 0.2.27, planned for tomorrow. Thanks for flagging this @brianpollack and @ireicht

**Comment by ireicht** [comment]
@yagil , thanks! Yes it is a duplicate of issue 42, no idea how I missed it. Guess I was to distracted by the football match yday which I watched in parallel :D 
FYI im on MacOS Sonoma 14.5 (M1 max and M3 max).

**Comment by yagil** [comment]
@brianpollack @ireicht here's 0.2.27: https://releases.lmstudio.ai/mac/arm64/0.2.27/latest/LM-Studio-0.2.27-arm64.dmg
please let me know if this solved the problem. Thanks!

**Comment by ireicht** [comment]
@yagil, excellent. Works like a charm. Thanks for the quick fix.
Keep on the great work.
